<launch>
  <arg name="plank_of_wood_id"/>
  <arg name="simulation"/>

  <include file="$(find gerard_bauzil)/launch/plank_of_wood.launch">
    <arg name="prefix" value="box" />
    <arg name="id" value="$(arg plank_of_wood_id)" />
  </include>
  <include file="$(find gerard_bauzil)/launch/rolling_table.launch">
    <arg name="prefix" value="table" />
  </include>
  <node name="to_tf_2" pkg="tf2_ros" type="static_transform_publisher"
    args="0 0 0 0 0 0 1 world odom"
    />

  <!-- Simulation -->
  <group if="$(arg simulation)" ns="simulation">
      <node name="simulation" pkg="agimus_sot" type="simulation.py" args="_tf_prefix:=" />
  </group>

</launch>
